docs/installation/integrations/next-js.md

```diff
@@ -0,0 +1,204 @@
+
+You can learn more about creating custom builds of CKEditor&nbsp;5 with the online builder in the {@link installation/getting-started/quick-start-other#creating-custom-builds-with-online-builder Customized installation} guide.
+
+## Setting up the project
+
+This guide assumes you will use the official CLI tool for Next.js &ndash; `create-next-app`. Refer to the [Next.js documentation](https://nextjs.org/docs/pages/api-reference/create-next-app) to learn how to set up your project.
+
+The CLI will ask you some questions. Depending on your choices, the folder structure in your project may differ.
+
+```plain
+What is your project named?  next-ckeditor-integration
+Would you like to use TypeScript?  No / Yes
+Would you like to use ESLint?  No / Yes
+Would you like to use Tailwind CSS?  No / Yes
+Would you like to use `src/` directory?  No / Yes
+Would you like to use App Router? (recommended)  No / Yes
+Would you like to customize the default import alias (@/*)?  No / Yes
+```
+
+### Page Router
+
+If you chose the Page Router, your folder structure should look similar to the one below.
+
+```plain
+├── pages
+│   ├── api
+│      └── hello.js
+│   ├── _app.js
+│   ├── _document.js
+│   └── index.js
+├── public
+│   ├── favicon.ico
+│   ├── next.svg
+│   └── vercel.svg
+├── styles
+│   ├── globals.css
+│   └── Home.module.css
+├── jsconfig.json
+├── next.config.js
+├── package.json
+└── ...
+```
+
+### App Router
+
+If you chose the App Router, your folder structure should look similar to the one below.
+
+```plain
+├── app
+│   ├── favicon.ico
+│   ├── globals.css
+│   ├── layout.js
+│   ├── page.js
+│   └── page.module.css
+├── public
+│   ├── next.svg
+│   └── vercel.svg
+├── jsconfig.json
+├── next.config.js
+├── package.json
+└── ...
+```
+
+## Integrating the build in your Next.js project
+
+With the Next.js setup done, let's move to the actual integration. First, put your unzipped custom build at the project root in the `ckeditor5` folder. Then, add it as a local dependency using the yarn command:
+
+```bash
+yarn add file:./ckeditor5
+```
+
+The dependency should be visible in the `package.json` file as `ckeditor5-custom-build`. Additionally, we need an external dependency &ndash; the CKEditor&nbsp;5 React component. Next.js extends React, and hence, the component is valid here. You can install it using the following command:
+
+```bash
+yarn add @ckeditor/ckeditor5-react
+```
+
+Next, we will use the installed dependencies in a React component. Create a new component in the components directory, for example, `components/custom-editor.js`. Inside the component file, import all necessary dependencies. Then, create a functional component that returns the CKEditor&nbsp;5 React component with a proper configuration.
+
+```jsx
+// components/custom-editor.js
+
+import React from 'react';
+import { CKEditor } from "@ckeditor/ckeditor5-react";
+import Editor from "ckeditor5-custom-build";
+
+const editorConfiguration = {
```

Comment:
Theoretically, this is not needed, as toolbar was configured in the online builder. I guess you want to show the possibility of extension. Maybe a comment mentioning this?

Reply:
Yeah, exactly. I'll add a comment.